src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py

```diff
@@ -134,7 +134,7 @@ def setJobParameter(self, jobID, key, value):
 
         self.log.debug("Inserting data in %s:%s" % (self.indexName, data))
```

Comment:
```suggestion
        self.log.debug("Inserting data", "in %s:%s" % (self.indexName, data))
```

Reply:
debug messages are not sent to ES, so it does not make sense to split between fixed and variable parts.